dove/tests/test_cmd_dove_clean.rs

```diff
@@ -0,0 +1,23 @@
+mod helper;
+use crate::helper::{execute_dove_at, project_start_new_and_build, project_remove};
+
+/// $ dove clean
+#[test]
+fn test_cmd_dove_clean() {
+    // Path to dove folder, project and project name
+    let project_name = "demoproject_16";
```

Comment:
Shouldn't we use human-readable directory name instead of random numbers?

Reply:
(of course, human readability is not necessary for temporary directories, but if we go this road and use distinct names, it's hard to stop here)